packages/components/forms/src/Select/Select.styles.ts

```diff
@@ -1,16 +1,29 @@
 import tokens from '@contentful/f36-tokens';
 import { css } from 'emotion';
 
-export function getSelectStyles({ isInvalid, isDisabled, size }) {
+export function getSelectStyles({ isInvalid, isDisabled, size, density }) {
+  const isHighDensity = density === 'high';
   const sizeStyles =
     size === 'small'
       ? {
           padding: `${tokens.spacing2Xs} ${tokens.spacingL} ${tokens.spacing2Xs} ${tokens.spacingXs}`,
           height: '32px',
+          fontSize: tokens.fontSizeM,
+          lineHeight: tokens.lineHeightM,
+          borderRadius: tokens.borderRadiusMedium,
         }
       : {
-          padding: `10px ${tokens.spacingL} 10px ${tokens.spacingS}`,
-          height: '40px',
+          padding: isHighDensity
+            ? `${tokens.spacingXs} ${tokens.spacingL} ${tokens.spacingXs} ${tokens.spacingXs}`
```

Comment:
I am a little concerned about the amount of inline ifs for this one variable. Would it not be easier to read and maintain to switch the whole class or write a "getStyles" sort of function? So we would have one if and avoid touching each and every single property

Reply:
It makes sense, adjusted 👍